data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,542 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	Anna shows you a webpage with all her stories. The viewer counts appear to be in the hundreds with the occasional one reaching four digits.`
+			choice
+				`	"So why is 'Beyond the Hyperdrive' about humans stealing from the Quarg?"`
+					goto "quarg"
+			label "book length"
+			`	"About three hundred pages. Not too long."`
+			choice
+				`	"Have you written any other novels?"`
+					goto "other novels"
+				`	"Why is it a story about humanity stealing from the Quarg?"`
+			label "quarg"
+			`	"I have a lot of problems with them. I just don't trust them. I remember when my parents took me to Humanika. Naturally, I wanted to explore, so I wandered off. I was lost. This creepy-looking Quarg found me, and when it tried to approach, I cried.`
+			`	"And then when I grew up, when I was a teenager, I decided to do a project on them for college. They responded with silence to my questions, as if they had something to hide. I worked as an assistant for a journalist for a few months. He kept asking me for help with the Quarg. They just would not budge."`
+			choice
+				`	"That must have been rough, stuck on Humanika. Did your parents eventually find you?"`
+					goto "parents"
+				`	"Well that's all well and good. But I have my own business to attend to now. Good luck with your book, thanks for your time, and goodbye."`
+					decline
+				`	(Tell her about the Coalition.)`
+					goto "coalition"
+			label "parents"
+			`	"They eventually found me, thankfully. But it was distressing being stuck with this Quarg for 10 to 30 minutes, I don't remember how long, before eventually this other Quarg took my parents to me. I told them I wanted to get off Humanika as fast as possible and get back to Zug. They obliged, and we were off that world and I never wanted to come back."`
+			choice
+				`	"So you said your book doesn't sell well, right?"`
+			`	"No, it doesn't. Do you know anyone who could help? My friends don't seem to be inclined to do anything for me."`
+			choice
+				`	"No actually, I'm afraid I don't know anyone who could."`
+					decline
+				`	(Tell her about the Coalition.)`
+			label "coalition"
+			`	You begin explaining the Coalition to Anna, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them. At first Anna does not believe you; but you reassure her it's real. You also mention the population of the Coalition and the novelty of an 'alien' author coming into the area.`
+			`	"Will they really like my books?"`
+			choice
+				`	"They will, hopefully."`
+				`	"Well, it's better to try than to just give up."`
+			`	"These stories might just inspire my next novel. But, they wouldn't speak our language. How are they going to read my books?"`
+			choice
+				`	"They have a translation device."`
+				`	"I'm sure they can learn the language."`
+			`	"That's reassuring... but, how am I even going to get there?" she queries.`
```

Comment:
It's best not to have choices interspersed with single lines of dialogue. Could you maybe cut one of these and just tell the player what they say? Something like "You explain that they have a translation device which allows them to communicate with humans."